go.sum

```diff
@@ -1,8 +1,32 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
+github.com/Azure/azure-pipeline-go v0.2.1/go.mod h1:UGSo8XybXnIGZ3epmeBw7Jdz+HiUVpqIlpz/HKHylF4=
```

Comment:
can we have a check about where these dependencies come from?

Reply:
```
github.com/ethereum/go-ethereum@v1.9.13 github.com/Azure/azure-pipeline-go@v0.2.2
```
ethereum depends on it.